DataHandler/.env

```diff
@@ -1,4 +1,4 @@
-SERVER=umfrage
+SERVER=localhost:1433
```

Comment:
localhost:1433 bitte durch $DATABASE_HOST ersetzen, env variable wird in der App-Compose.yml gesetzt und gibt dir die Addresse zur Datenbank